src/improvements/tasks/MultisigTask.sol

```diff
@@ -612,14 +653,17 @@ abstract contract MultisigTask is Test, Script {
     function printNestedDataToSign() public view {
         bytes memory callData = generateApproveMulticallData();
 
-        if (childMultisig != address(0)) {
+        if (childMultisig != address(0) && childMultisig.code.length != 0) {
```

Comment:
Should this be replaced with `if (isNestedSafe)`? If not, what's the difference between that bool and this check? Code comments explaining the "why" behind logic would be helpful here

Reply:
> if `isNestedSafe` is true, that does not imply that `childMultisig` address is non zero. if `childMultisig` address is non zero, that does not imply anything about whether or not this is a nested safe, as a child multisig can have other Gnosis Safe signers.

Why do we care if a child multisig can have other safe signers? More generically, if took the implementer of this code running tests to realize that `childMultisig != address(0)` is not equivalent to `isNestedSafe`, the code is probably confusing and we should make sure to add code comments or helper methods to make things more clear :)